popgen/mcmcSelection.go

```diff
@@ -154,3 +180,16 @@ func findMaxN(data AFS) int {
 	}
 	return answer
 }
+
+func cacheIndexToAlleleFrequencyCache(bins int) []float64 {
+	var answer []float64 = make([]float64, bins + 1)
+	//this is done with linear interpolation, as index zero corresponds to an allele frequency of the LeftBound and the index at len[n][k], the number of values p, is the rightBound allele frequency. y = mx + b. First we find m and b.
+	//b is simply the LeftBound.
+	m := (LeftBound - RightBound) / (bins + 1)
```

Comment:
Where do LeftBound and RightBound come from?  Should it be RightBound - LeftBound?  I don't see them defined in this function or this file.  Maybe global to this package and defined in another file?  Should (bins + 1) be "bins"?